go/libraries/doltcore/sqle/dolt_index.go

```diff
@@ -141,26 +145,31 @@ func (di *doltIndex) Get(keys ...interface{}) (sql.IndexLookup, error) {
 	readRange := &noms.ReadRange{Start: tpl, Inclusive: true, Reverse: false, Check: func(tuple types.Tuple) (bool, error) {
 		return tuple.StartsWith(tpl), nil
 	}}
-	return di.rangeToIter(readRange)
+	return di.rangeToIndexLookup(readRange)
 }
 
+// Has implements sql.Index
 func (*doltIndex) Has(partition sql.Partition, key ...interface{}) (bool, error) {
 	// appears to be unused for the moment
 	panic("implement me")
 }
 
+// ID implements sql.Index
 func (di *doltIndex) ID() string {
 	return di.id
 }
 
+// Schema returns the dolt schema of this index.
 func (di *doltIndex) Schema() schema.Schema {
 	return di.tableSch
 }
 
+// Table implements sql.Index
 func (di *doltIndex) Table() string {
 	return di.tableName
 }
 
+// TableData returns the map of table data for this index (the map of the index table, not the target table)
```

Comment:
The comment is backwards, `di.tableData` is the target table's data. `di.indexRowData` is the map of the index table.